docs/sidebars.js

```diff
@@ -49,6 +49,7 @@ const sidebars = {
           items: [
             "guides/authentication/custom-auth-upstream",
             "guides/authentication/oauth2",
+            "guides/authentication/cloudentity",
```

Comment:
let's not add it in the sidebar until they're happy.. 
but we also don't have any doc 🤔 https://github.com/kubeshop/kusk-gateway/pull/796/files#diff-2e24c757c8081d933c0654a34501fa6ebe0a240f4e205f4bafa04d176fae16ecR1

Reply:
I've changed it to coming soon.

Is that OK?

Reply:
No just remove it until we get the green light to publish it. There are several more steps to announce this.

Reply:
Removed in commit <https://github.com/kubeshop/kusk-gateway/pull/796/commits/b7a27035ffdda8721e0c2c5bfbc57e08083a9e65>.